### ISSUE TITLE
Grafana  DashBoard K8s Pod Map missing edges under some query conditions

### ISSUE THREAD

**Issue by liudicust** [deepflowys/deepflow#1115]
# Actual Behavior
图中标红色的链接（cumulx-977666c9d-ghq5t 与 数据库 10.246.97.121）在拓扑图中并没有展示出来，但是在Pod path面板中是存在的，实际上也是存在这条链接的
![09027D2E-0439-432f-AA23-F3DDFB7D6537](https://user-images.githubusercontent.com/8587857/196867900-b157fc89-b790-4e15-95ab-33602bcd0da9.png)

# Steps to Reproduce the Problem
Grafana  DashBoard K8s Pod Map
检索条件：某个namespace，任意时间段（last 5 minites等）都会出现

# Additional Info
每次丢失的连接线不一定是哪条，也有可能是0.0.0.0
-  deepflow version:

```
deepflowce/grafana:9.1.7
deepflowce/deepflow-server:v6.1.4
deepflowce/deepflow-agent:v6.1.4
```

- deepflow agent list:

  **Output of `deepflow-ctl agent list`:**
```
dev-worker-1-V9 K8S_VM     10.246.97.49     00:16:3e:00:05:bb  RUNNING             default
dev-master-2-V8 K8S_VM     10.246.97.47     00:16:3e:00:07:20  RUNNING             default
dev-worker-5-V3 K8S_VM     10.246.97.53     00:16:3e:00:0a:27  RUNNING             default
dev-master-3-V7 K8S_VM     10.246.97.48     00:16:3e:00:02:40  RUNNING             default
dev-master-1-V1 K8S_VM     10.246.97.46     00:16:3e:00:01:86  RUNNING             default
dev-worker-3-V4 K8S_VM     10.246.97.51     00:16:3e:00:09:65  RUNNING             default
dev-worker-4-V5 K8S_VM     10.246.97.52     00:16:3e:00:05:a1  RUNNING             default
dev-worker-6-V6 K8S_VM     10.246.97.54     00:16:3e:00:01:2b  RUNNING             default
dev-worker-2-V2 K8S_VM     10.246.97.50     00:16:3e:00:04:91  RUNNING             default
```

- Kubernetes CNI:

```
flannel/vxlan
```

- operation-system/kernel version:
  
  **Output of `awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release`:**
  
  **Output of `uname -r`:**
```
CentOS Linux 7 (Core)
4.19.12-1.el7.elrepo.x86_64
```

**Comment by twou12031** [comment]
正在尝试复现, 并尽快修复此问题.
We're working to reproduce the bug and fix it as quickly as possible.

**Comment by twou12031** [comment]
这个问题已经被修复, 你可以使用 v6.1.5 版本查看修复后的结果.
谢谢你的反馈.
This bug has been fixed, you could use the version v6.1.5 to see it.
Thank you for your feedback.
